Warsaw_Tram/Tram.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TramViewModel.swift
+//  Warsaw_Tram
+//
+//  Created by Małgorzata Dziubich on 26/06/16.
+//  Copyright © 2016 Małgorzata Dziubich. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+
+class Tram {
+
+    var number: String!
+    var lattitude: CLLocationDegrees!
+    var longitude: CLLocationDegrees!
+    var lowFloor: Bool!
+    var status: String!
```

Comment:
This is wrong. To note that a parameter can not be `nil` type is just enough. When you use `!` you are saying to the compiler that I don't what to set parameter during object initialisation but you are sure that when you use them first time there will be some value (not nil). You are setting this values in init, I'd rather remove `!`. You can read more about `implicitly unwrapped` and `optional paramter`